Sources/HBLambdaTest/maths.swift

```diff
@@ -40,25 +50,32 @@ struct MathsHandler: HBLambda {
         let result: Double
     }
 
-    init(_ app: HBApplication) {
-        app.encoder = JSONEncoder()
-        app.decoder = JSONDecoder()
-        app.middleware.add(DebugMiddleware())
-        app.router.post("add") { request -> Result in
-            let operands = try request.decode(as: Operands.self)
+    let router: HBRouterBuilder<Context>
```

Comment:
You don't need to store the router here. You only need to set the responder at `init`. Currently you are building the responder every time a request comes through

Reply:
That's not what happens, because we store the responder in the LambdaHandler.

Reply:
The problem arises because there's no public type or generic type that represents the Responder of an HBRouterBuilder. Therefore, I cannot annotate the type of responder here. So in order to prevent building a new responder for each request, I made sure to cache the value in HBLambdaHandler.

Even if we remove `some HBResponder<Context>` from `HBRouterBuilder` by making `HBRouter` public - I think this is still a usecase that will pop up more often. In particular when your new router builder PR is merged.